<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapesAndLayersVisibilityHelperDialog</class>
 <widget class="QDialog" name="ShapesAndLayersVisibilityHelperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShapesAndLayers Visibility Helper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbBlockInvisibileLayer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Soft block and warn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Block &amp; make exception for current layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Block &amp; go up if current layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Block &amp; go down if current layer</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cmbToggleVisibilityDrag">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>same visibility</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>invert visibility</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="boolBlockInvisibileLayer">
     <property name="text">
      <string>Block invisible layers</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="boolToggleVisibilityDrag">
     <property name="text">
      <string>Toggle visibility drag</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="boolAutoSelectVisibleLayer">
     <property name="text">
      <string>Auto-select layer on making layer visible</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShapesAndLayersVisibilityHelperDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShapesAndLayersVisibilityHelperDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
